src/AWSClusterManagersDemo.jl

```diff
@@ -0,0 +1,44 @@
+using AWSClusterManagers: AWSBatchManager
+using Distributed
+
+const NUM_WORKERS = 4
+
+# Must start workers before any @everywhere for them to be available on the workers
+addprocs(AWSBatchManager(NUM_WORKERS))
+
+@everywhere function throw_darts()
+    MAX_THROW = 250_000
+    dart_count = Integer(floor(rand() * MAX_THROW))
+    hits = 0
+
+    for _ in 1:dart_count
+        x = rand()
+        y = rand()
+
+        if ((x*x) + (y*y) < 1)
+            hits += 1
+        end
+    end
+
+    return hits, dart_count
+end
+
+calc_approx_pi(hits::Integer, thrown::Integer) = 4 * hits / thrown
```

Comment:
if using random number between -1 and 1 then don't need this?

Reply:
Which would be nice since then would see each worker's approximation of pi?

Reply:
This comes from the article which you've linked me, [here](https://jccraig.medium.com/calculate-pi-with-a-dartboard-bdb433f1c999).

Basically, there are four workers, each throw up to 250k darts. The amount each throws and their hits are recorded and output, then combined together to calculate ~PI.

I just took the Python script from the link above and re-wrote it in Julia.

Reply:
I make no promises about the quality of that article though.
I just wanted to express the idea.

Reply:
Hehe, it seemed fine enough and a decent example to implement here!